README.md

```diff
@@ -21,3 +21,29 @@ In order to open traces you need a trace server running on the same machine as t
 2. From the repo root:  `yarn start:browser` or `yarn start:electron`
 3. Go to http://localhost:3000 or use the Electron application
 
+## Using the trace extension
+This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX is prelimiary and subject to revisions in the future.
+
+### Open trace(s)
+To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With-> Open Trace(s)**.
+
+If the selection is a single file, then the tool will open the file directly as an experiment.
+
+If the selection is a directory, then tool will look for traces in **Common Trace Format (CTF)** format, such as **Linux Tracing Toolkit traces (LTTng)** Kernel and UST (Userspace) traces, and open all found CTF traces as an experiment. With this you're able to open e.g. LTTng Kernel and UST Traces at the same time.
```

Comment:
then the* tool

Does it open only CTF traces? or any trace type supported by the server? Is the issue server side or theia side? If only CTF traces can be opened, you should open an issue to track this and add support or any trace type supported by server.

Reply:
If it's a single file (e.g. jaeger traces in json) and the server supports that trace type it will open it as an experiment. Opening multiple traces from a root directory will only open CTF traces found in the that directory (recursively). Any other files will be ignored because the front-end has no knowledge of trace types and how to dedect them. The trace detection is another topic that we need to handle in the future (e.g. maybe a trace detection service ?)